packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.stories.tsx

```diff
@@ -11,34 +14,29 @@ export default {
 
   component: YppAuthorizationModal,
   argTypes: {
-    step: {
-      options: ['select-channel', 'requirements', 'details', 'terms-and-conditions', 'summary', 'connect-with-yt'],
-      control: { type: 'select' },
+    currentStepIdx: {
+      type: 'number',
     },
-    channels: { table: { disable: true } },
   },
   args: {
-    show: true,
-    step: 'requirements',
-    channels: [
-      {
-        id: '1',
-        title: 'The Joystreamers',
-        createdAt: new Date(),
-        follows: 12332,
-        rewardAccount: '',
-        channelStateBloatBond: '',
-      },
-      {
-        id: '2',
-        title: 'Joystream',
-        createdAt: new Date(),
-        follows: 987,
-        rewardAccount: '',
-        channelStateBloatBond: '',
-      },
-    ],
+    currentStepIdx: null,
```

Comment:
Can we add the `select` control here, or something more descriptive? 
![image](https://user-images.githubusercontent.com/51168865/193542928-c7fd2a8c-9b9a-48b2-8abb-8c0ee6537fed.png)
Right now I'm not sure what I can type here. I mean, I can change the number, but in the end, I don't know what a particular number stands for.

Reply:
Fixed